admin-core/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM eclipse-temurin:11.0.21_9-jdk-nanoserver
+
+RUN mkdir distr
```

Comment:
Each docker command creates a separate docker layer (with its own cache) - avoid spamming RUNs -> prefer to use `&&` if commands belong to the same logical "step":
```
RUN  mkdir && \
mkdir work
```
`&& \` - is equivalent to `&&`. Backslash `\` escapes line return symbol, so visually separate lines will  be processed as a single line

Reply:
done